src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -900,4 +900,16 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="Accessibility_Packages" xml:space="preserve">
     <value>Packages</value>
   </data>
+  <data name="Status_Canceled" xml:space="preserve">
+    <value>Package results loading cancelled</value>
+  </data>
+  <data name="Status_ErrorOccurred" xml:space="preserve">
+    <value>Error occurred when loading package results</value>
+  </data>
+  <data name="Status_NoMoreItems" xml:space="preserve">
+    <value>Last package results page loaded</value>
```

Comment:
"Last" isn't clear. Try a more clear word like "Completed loading of packages" as my first thought.

Reply:
Tried to express that the list is paginated.

Reply:
I prefer replacing "Last" with something like "Final".

Reply:
What about `End of package list loaded` ?

Reply:
It's not great but more clear than the current value. Get input from others if possible

Reply:
I will do that later. Tracking isue: https://github.com/NuGet/Home/issues/9445